examples/BERT/README.md

```diff
@@ -0,0 +1,56 @@
+# BERT with torchtext
+
+This example shows how to train a BERT model with PyTorch and torchtext only. Then, we fine-tune the pre-trained BERT for the question-answer task.
+
+
+Generate pre-trained BERT
+-------------------------
+
+Train the BERT model with masked language modeling task and next-sentence task. Run the tasks on a local GPU or CPU:
+
+    python mlm_task.py
+    python ns_task.py
+
+or run the tasks on a SLURM powered cluster with Distributed Data Parallel (DDP):
+
+    srun --label --ntasks-per-node=1 --time=400 --mem-per-cpu=5120 --gres=gpu:8 --cpus-per-task 80 --nodes=1 --pty python mlm_task.py --parallel DDP
```

Comment:
We should make an explicit decision on whether we should add SLURM support or not. Since SLURM is only one possible technology that users could use, I would lean for removing this from the implementation to keep things leaner. Thoughts?

I only see partial references in torchvision.
```
classification/utils.py:    elif 'SLURM_PROCID' in os.environ:
classification/utils.py:        args.rank = int(os.environ['SLURM_PROCID'])
detection/utils.py:    elif 'SLURM_PROCID' in os.environ:
detection/utils.py:        args.rank = int(os.environ['SLURM_PROCID'])
segmentation/utils.py:    elif 'SLURM_PROCID' in os.environ:
segmentation/utils.py:        args.rank = int(os.environ['SLURM_PROCID'])
video_classification/utils.py:    elif 'SLURM_PROCID' in os.environ:
video_classification/utils.py:        args.rank = int(os.environ['SLURM_PROCID'])
```

Reply:
In the specific task, like `ns_task.py` and `mlm_task.py`, we support single machine (either GPU or CPU) and DDP. This command here is to show how to run DDP with SLURM. The implementation of the tasks doesn't assume SLURM as the distributed technology.  @vincentqb